testing/src/main/java/net/consensys/linea/testing/ToyExecutionEnvironment.java

```diff
@@ -254,7 +254,7 @@ private MainnetTransactionProcessor getMainnetTransactionProcessor() {
         new MessageCallProcessor(evm, precompileContractRegistry);
 
     final ContractCreationProcessor contractCreationProcessor =
-        new ContractCreationProcessor(evm, false, List.of(), 0);
+        new ContractCreationProcessor(evm, false, List.of(), 1);
```

Comment:
I implemented the same fix on my working branch for the `ToyExecutionEnvironmentV2` BTW. It's not solving our issues with deployments.